### PR TITLE
DR-801 Change default GCR tag to getGitHash.

### PR DESCRIPTION
**Pull request by marikomedlock**
Changed the default container tag to use the most recent git commit hash, instead of latest. The way to specify a custom container tag is unchanged -- set the GCR_TAG environment variable. Also added the workaround for a bug/feature in the Jib Grade plugin that automatically adds the latest tag if no other tag is specified.

Links from @smark88 about the Jib Gradle plugin bug/feature. Workaround is at the bottom of the second link.
https://github.com/GoogleContainerTools/jib/issues/1226
https://github.com/GoogleContainerTools/jib/issues/1427

I tested this locally with these commands, and checked the tags in GCR console:
./gradle :jib
GCR_TAG=marikoTESTING ./gradlew :jib

### CODE REVIEW

**Review by smark88 — APPROVED** [review]

File: src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java [hunk 1]
```diff
@@ -106,9 +106,9 @@ public FSFileInfo copyFile(Dataset dataset,
             // so I changed exported the timeouts to application.properties to allow for tuning
             // and I am changing this to copy chunks.
             CopyWriter writer = sourceBlob.copyTo(BlobId.of(bucketResource.getName(), targetPath));
-            while (!writer.isDone()) {
```
Comment (fboulnois) [inline src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java]: Is this change necessary for the gradle change?

**Review by fboulnois — CHANGES_REQUESTED** [review]

File: src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java (hunk 1 above)
Comment (marikomedlock) [reply src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java]: No it's not supposed be a part of this PR. I meant to push this commit to another branch. Thanks for catching it!

**Review by myessail — APPROVED** [review]

**Review by fboulnois — APPROVED** [review]
lgtm
